docs/known-issues.md

```diff
@@ -0,0 +1,35 @@
+
+# Known Issues and Limitations
+
+## Language systems and libraries
+* CPU affinity API is silently ignored
+* Language runtime base images are provided for a single version per language,
+  e.g. for Python 3.8 and not for Python 2 or Python 3.9

```

Comment:
Add period to end of this line.

Add a sentence "Please contact us if you need a different version."